specificatie/openapi.yaml

```diff
@@ -82,6 +83,7 @@ paths:
         - $ref: '#/components/parameters/pandIdentificatie-query'
         - $ref: '#/components/parameters/adresseerbaarObjectIdentificatie-query'
         - $ref: '#/components/parameters/zoekresultaatIdentificatie'
+        - $ref: '#/components/parameters/zoek'
```

Comment:
De beschrijving voor de zoek functionaliteit uit de description bij /adressen/zoek, moet hier worden opgenomen.

Reply:
@JohanBoer deze zijn we vergeten over te nemen!
Ik zou er zoiets van maken:

"Vind een actueel adres met:
1. een pand identificatie of 
2. een adresseerbaar object identificatie of
3. een zoekterm op postcode, woonplaats, straatnaam, huisnummer, huisletter, huisnummertoevoeging. 

Gebruik de fields-parameter als je alleen specifieke velden in het antwoord wil zien, zie [functionele specificaties fields-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/fields.feature). Gebruik de expand-parameter als je het antwoord wil uitbreiden met (delen van) de gerelateerde resources nummeraanduiding, woonplaats en openbare ruimte, zie [functionele specificaties expand-parameter](https://github.com/VNG-Realisatie/Haal-Centraal-common/blob/v1.1.0/features/expand.feature)."